packages/flutter/test/widgets/navigator_test.dart

```diff
@@ -4231,14 +4231,14 @@ void main() {
     }
 
     await tester.pumpWidget(build());
-    observer._checkInvocations(<Symbol>[#navigator, #didPush]);
+    observer._checkInvocations(<Symbol>[#navigator, #didPush, #didChangeTop]);
     await tester.pumpWidget(Container(child: build()));
-    observer._checkInvocations(<Symbol>[#navigator, #didPush, #navigator]);
+    observer._checkInvocations(<Symbol>[#navigator, #didPush, #didChangeTop]);
     await tester.pumpWidget(Container());
-    observer._checkInvocations(<Symbol>[#navigator]);
```

Comment:
It wasn't immediately clear to me why the `#navigator` was removed from these invocations. That's intentional?